IGNITE-14247 Add snapshot page, add checking snapshot commands (#8878)
apache/ignite: docs/_docs/snapshots/snapshots.adoc

----
--

== Checking Snapshot Consistency

> Usually all the cluster nodes are running on different machines and having the snapshot data spread across the cluster.

Reviewer: ```suggestion
Usually all the cluster nodes run on different machines and have the snapshot data spread across the cluster.
```

Each node stores its own snapshot segment, so in some cases it may be necessary to check the snapshot for completeness
of data and for data consistency across the cluster before restoring from the snapshot.

> For such cases, the Apache Ignite ships with the built-in snapshot consistency check commands that enable you to verify

Reviewer: ```suggestion
For such cases, Apache Ignite is delivered with built-in snapshot consistency check commands that enable you to verify
```

> internal data consistency, calculate data partitions hashes and pages checksums, and printing out the result if a

Reviewer: ```suggestion
internal data consistency, calculate data partitions hashes and pages checksums, and print out the result if a
```
